staging/src/k8s.io/apiserver/plugin/pkg/authenticator/token/webhook/webhook_v1beta1_test.go

```diff
@@ -0,0 +1,686 @@
+/*
```

Comment:
this file was a rename of webhook_test.go with V1beta1 added to type/function names